ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -260,6 +260,12 @@ By default an MLAG ID will only be configured on Port-Channel downlinks dual-hom
 + mlag_on_orphan_port_channel_downlink: true
 ```
 
+### Raise the AVD error if `sFlow` is enabled on an interface but `sflow_settings.destinations` is not defined
+
+In AVD 4.0.0, if `sFlow` was enabled on an interface but `sflow_settings.destinations` was not defined, the sFlow configuration was not generated.
```

Comment:
```suggestion
In AVD 4.0.0, if `sflow` was enabled on an interface but `sflow_settings.destinations` was not defined, the sFlow configuration was still generated for interfaces, although it would not work without "custom structured configuration".
```